modules/caseyduquettesc_rules_python_pytest/1.1.1/presubmit.yml

```diff
@@ -0,0 +1,16 @@
+bcr_test_module:
+  module_path: "e2e/smoke"
+  matrix:
+    # Disable debian tests because I don't think it's using the python toolchain,
+    # > no such package '@[unknown repo 'pip_importlib_metadata' requested from @rules_python~0.19.0~pip~pip_pytest]//'
+    # platform: ["debian10", "macos", "ubuntu2004"]
+    platform: ["macos", "ubuntu2004"]
+    bazel:
+      - 6.x
```

Comment:
nit: does this module support bazel 7.x?

Reply:
It might, I just haven't been able to test it out yet.